content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

```diff
@@ -1,35 +1,33 @@
-=======================
-Variants BOM Management
-=======================
+=============================
+Managing product variant BoMs
+=============================
 
-Odoo allows you to use one bill of materials for multiple variants of
-the same product. To activate variants, simply go to :menuselection:`Configuration --> Settings
---> Variants`.
+In Odoo, one bill of material can be used for multiple variants of the same product. Having a
+consolidated BoM for a product with variants saves time by preventing the need to manage multiple
+BoMs. To activate variants, simply navigate to :menuselection:`Inventory --> Configuration --> Settings --> Variants`. For more information on configuring product variants, refer to this page. 
 
 .. image:: product_variants/bom-variants1.png
-    :align: center
+   :align: center
+   :alt: 
 
-You will then be able to specify which BOM component lines are to be used in
-the manufacture of each product variant. You may specify multiple
-variants for each line. If no variant is specified on a component line,
-then this component will be used in the BOM of all variants. 
-
-When defining variant BoMs on a line-item-basis, the **Product
-Variant** field in the main section of the BoM should be left blank as shown 
-below. This field is used when creating a BoM for one specific variant
-of a product only.
+Once product variants are fully configured, components may be assigned to specific variants on the
+bill of materials. Each component may be assigned to multiple variants. Components with no variant
+specificed are used in every variant of the product.
 
+When defining BoMs for product variants by component assignment, the :guilabel:`Product Variant`
+field in the main section of the BoM should be left blank, as shown below. This field is used only
+when creating a BoM for one specific product variant.
 
 .. image:: product_variants/kit-bom1.png
-    :align: center
+   :align: center
 
 Note that the same principle applies for the configuration 
 of operations.
 
 .. image:: product_variants/kit-bom2.png
-    :align: center
+   :align: center
 
 And for the production of by-products. 
```

Comment:
Lines 24/25 and line 30 are fragments of the same sentence.

Ideally these get combined into one sentence OR more/complete context is added to both paragraphs in order to strengthen the written documentation and + break the dependency on the images to explain what's going on.